README.md

```diff
@@ -192,6 +192,10 @@ The `-h` flag above (shorthand of `--help`) defines that setup script must only
 - `bin/setup/xdebug`: Enable remote debugging via set of xdebug.* ini settings in .user.ini file. Please see the detailed instructions of how to configure and use it on our Wiki page [Xdebug & PHPStorm](https://github.com/mage2click/docker-magento-mutagen/wiki/Xdebug-&-PHPStorm) 
 ## Misc Info
 
+### Admin Panel
+The admin panel URL is auto-generated by the setup script. You can find the URL in the output in your terminal. The default username is `john.smith` and the default password is `password123`. You are free to change these default values in the following file:
```

Comment:
@magedevjosh, thanks for PR!

Would be good to add here command `bin/magento info:admin` which prints admin URL instead of searching admin URL in installation log.

Reply:
I always forget about that command. Good catch. Updated.